app/registries/foreman/settings/general.rb

```diff
@@ -32,8 +32,8 @@
       full_name: N_('Smart Proxy request timeout'))
     setting('login_text',
       type: :text,
-      description: N_("Text to be shown in the login-page footer"),
-      default: nil,
+      description: N_("Text to be shown in the login-page footer. Keyword $VERSION is replaced by current version."),
+      default: N_("Version $VERSION"),
```

Comment:
Maybe a nitpick, it is more translator-friendly to do this: `N_("Version") + "$VERSION",`